projects/packages/my-jetpack/_inc/components/my-jetpack-screen/index.jsx

```diff
@@ -125,8 +126,10 @@ export default function MyJetpackScreen() {
 	const [ reloading, setReloading ] = useState( false );
 
 	useEffect( () => {
-		recordEvent( 'jetpack_myjetpack_page_view' );
-	}, [ recordEvent ] );
+		recordEvent( 'jetpack_myjetpack_page_view', {
+			red_bubble_alerts: Object.keys( redBubbleAlerts ),
```

Comment:
I think Tracks has trouble with arrays. 
We could either JSON encode this or maybe implode the array with a separator like `|` so that we could match for specific values?